### PR TITLE
use lowercase for quadbin values in SF

### PR DESCRIPTION
**Pull request by volaya**
https://app.shortcut.com/cartoteam/story/241885/tileset-spatial-index-snowflake-quadbin-the-widgets-don-t-work-with-tilesets

### CODE REVIEW

**Review by padawannn — CHANGES_REQUESTED** [review]
It only solves the problem for quadbin, right? I've also detected the same problem for h3

**Comment by Jesus89** [comment]
For H3, let's add the functions in the h3 module
- _H3_INT_TOSTRING
- _H3_STRING_TOINT

And then use them in the tiler

**Comment by Jesus89** [comment]
Yes, but let's move the implementation of these functions to the core: https://github.com/CartoDB/analytics-toolbox-core/pull/264#issuecomment-1175911007. So we have one common unified implementation with also corresponding tests.

After merging this, update core submodule in the branch for the tiler to use the new functions

**Comment by volaya** [comment]
@Jesus89 The PR already includes the H3 functions and corresponding tests. Let me know when you can review it, so it can be merged and I can update the AT repo (I have that PR already finished, waiting for this to be merged)

File: modules/h3/bigquery/sql/H3_INT_TOSTRING.sql
Comment (Jesus89) [inline modules/h3/bigquery/sql/H3_INT_TOSTRING.sql:7]: You can rewrite this as `FORMAT('%x', h3)`

**Review by Jesus89 — APPROVED** [review]
LGTM

Suggested an optimization for H3_INT_TOSTRING

Complete these pending updates before merging:
- Update common/bigquery/version
- Update modules/h3/bigquery/CHANGELOG.md
- Update modules/h3/bigquery/package.json
- Update common/snowflake/version
- Update modules/h3/snowflake/CHANGELOG.md
- Update modules/h3/snowflake/package.json

**Pull request merged**
